apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/LoggingUtils.java

```diff
@@ -60,6 +60,15 @@ public static void logBenesToMdc(org.hl7.fhir.r4.model.Bundle bundle) {
     logBeneIdToMdc(beneIds.stream().toArray(Long[]::new));
   }
 
+  /**
+   * Log resource count returned to client given size of single element or bundle to BfdMDC.
+   *
+   * @param count of resources returned to client to log
+   */
+  public static void logResourceCountToMdc(int count) {
+    BfdMDC.put("resource_count", String.format("%d", count));
```

Comment:
can we call this resources_returned_count? (to disambiguate it from all of the other keys that have the word resource in them which are just resources queried and not necessarily returned)

Reply:
Updated